src/app/reviews/review-search/review-search.component.html

```diff
@@ -0,0 +1,33 @@
+<form (ngSubmit)="onSearchForReview()">
+  <div class="inner-form">
+      <div class="input-field first-wrap">
+        <div class="svg-wrapper">
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
please move to svg code to a svg file . you can use background image in css to display it.

Reply:
well I can't because if I click on the link it is impossible to download as save it as a file